### ISSUE TITLE
Not valid/missing advanced values @ data['options']

### ISSUE THREAD

**Issue by lsilva171** [danielwelch/hassio-zigbee2mqtt#184]
**zigbee2mqtt add-on version (if edge, please report commit hash):**
1.5.1

**Operating environment (HassOS, Virtual Machine, Device/platform running Hass.io):** 
Hass.io 0.96.2 on Odroid XU4 running Ubuntu 18.04

**Description of problem:**
Advanced configuration entries seem to be missing. Adding last_seen or elapsed to advanced generates an error when saving configuration saying not a valid value for dictionary value @ data['options'].

**Your entire configuration from the frontend (with sensitive fields redacted)**:
```json
{
  "data_path": "/share/zigbee2mqtt",
  "devices": "devices.yaml",
  "groups": "groups.yaml",
  "homeassistant": true,
  "permit_join": false,
  "mqtt": {
    "base_topic": "zigbee2mqtt",
    "server": "mqtt://redacted",
    "user": "redacted",
    "password": "redacted"
  },
  "serial": {
    "port": "/dev/ttyACM0"
  },
  "advanced": {
    "pan_id": 6754,
    "channel": 11,
    "last_seen": "enable",
    "elapsed": true,
    "network_key": [redacted]
  }
}

**Comment by dzungpv** [comment]
I add it here please waiting for update https://github.com/danielwelch/hassio-zigbee2mqtt/pull/185

**Comment by danielwelch** [comment]
this should be fixed thanks to dzungpv in latest master and v1.5.1.1. Please re-open if you run into more related problems.
